### ISSUE TITLE
Installation issue when doing initial migrate

### ISSUE THREAD

**Issue by devran** [django-wiki/django-wiki#84]
Hi,

I was doing a fresh install and at the step:
python manage.py migrate
...I got this this output:

---- SNIP ---
  File "/home/dev/django/lib/python2.7/site-packages/django/db/backends/postgresql_psycopg2/base.py", line 52, in execute
    return self.cursor.execute(query, args)
## django.db.utils.DatabaseError: relation "notify_subscription" does not exist

For anyone else in the same situation, the problem was solved by first doing this:
python manage.py migrate django_notify
and then:
python manage.py migrate

In other words, there seems to be some error in the installation order. Not a big thing, more FYI :)

Thanks for your hard work,
Devran
